app/metrics.yaml

```diff
@@ -178,6 +178,30 @@ browser.search:
       - android-probes@mozilla.com
     expires: "2022-07-01"
 
+  search_count:
+    type: labeled_counter
+    description: |
+      The labels for this counter are `<search-engine-name>.<source>`.
+
+      If the search engine is bundled with Focus `search-engine-name` will be
+      the name of the search engine. If it's a custom search engine (defined:
+      https://github.com/mozilla-mobile/fenix/issues/1607) the value will be
+      `custom`.
+
+      `source` will be: `action`, `suggestion`
```

Comment:
From I saw [here](https://dictionary.telemetry.mozilla.org/apps/fenix/metrics/metrics_search_count) and in Fenix, action and suggestions aren't the only possible sources, but in the old telemetry only those are treated
I will make a new ticket and investigate the support for others

Reply:
https://github.com/mozilla-mobile/focus-android/issues/6252